app/tests/integration/nimbus/pages/browser/__init__.py

```diff
@@ -7,12 +7,12 @@
 
 
 class Browser:
-    def execute_script(selenium, *args, script=None, context=None):
+    def execute_script(self, *args, script=None, context=None):
         if "chrome" in context:
-            with selenium.context(selenium.CONTEXT_CHROME):
-                return selenium.execute_script(script, *args)
+            with self.context(self.CONTEXT_CHROME):
+                return self.execute_script(script, *args)
         else:
-            return selenium.execute_script
+            return self.execute_script
```

Comment:
I don't think this will work

Reply:
Wow, so this passed. I guess it is technically ```self``` then....huh...

Reply:
The first argument to a class method is always the instance of the class the method is being called on, it can be called anything but it'll always point to that, so conventionally it's called 'self':

https://docs.python.org/3/tutorial/classes.html

```
Often, the first argument of a method is called self. This is nothing more than a convention: the name self has absolutely no special meaning to Python. Note, however, that by not following the convention your code may be less readable to other Python programmers, and it is also conceivable that a class browser program might be written that relies upon such a convention.
```

Reply:
Ah so this "self" is from the original class I used to call this function. Makes sense!